src/model/generated/rentalStarted.model.ts

```diff
@@ -1,9 +1,11 @@
-import {Entity as Entity_, Column as Column_, PrimaryColumn as PrimaryColumn_, Index as Index_} from "typeorm"
+import {Entity as Entity_, Column as Column_, PrimaryColumn as PrimaryColumn_, Index as Index_, OneToMany as OneToMany_} from "typeorm"
 import * as marshal from "./marshal"
+import {Item} from "./item.model"
+import {Hook} from "./hook.model"
 
 @Entity_()
-export class RentalManagerEventRentalStarted {
-    constructor(props?: Partial<RentalManagerEventRentalStarted>) {
+export class RentalStarted {
+    constructor(props?: Partial<RentalStarted>) {
         Object.assign(this, props)
     }
```

Comment:
So, would it be possible to let class instantiation handle the `hooks` and `items`?

```suggestion
export class RentalStarted {
    constructor(props?: Partial<RentalStarted>) {
        Object.assign(this, props);
        
        this.items = props.items.map((item) => ({
          ...item,
          id: randomUUID(),
          rentalStarted: this,
        }));
        
        this.hooks = props.items.map((hook) => ({
          ...hook,
          id: randomUUID(),
          rentalStarted: this,
        }));
    }
```

In the `create` mapping you then could instantiate like so?

```
new RentalStarted({
  ...
  items,
  hooks,
  ...
});
```

Unsure if TS would play along—I think it should. Nitpicking woo!